public/pages/CreateMonitor/components/AssociateMonitors/components/MonitorsList.js

```diff
@@ -0,0 +1,286 @@
+          name={formikFieldName}
+          form={form}
+          rowProps={{
+            label: 'Monitor',
+            isInvalid: () => form.touched[formikFieldName] && !isValid(),
+            error: () => validate(),
+          }}
+        >
+          <Fragment>
+            {fields.map((monitorIdx, idx) => (
+              <EuiFlexGroup
+                key={`monitors_list_${monitorIdx}`}
+                style={{ width: '400px', position: 'relative' }}
+              >
+                <EuiFlexItem grow={true}>
+                  <FormikComboBox
+                    name={`${formikFieldName}_${monitorIdx}`}
+                    inputProps={{
+                      isInvalid:
+                        (form.touched[`${formikFieldName}_${monitorIdx}`] ||
+                          form.touched[formikFieldName]) &&
+                        !selection[monitorIdx],
+                      placeholder: 'Select a monitor',
+                      onChange: (options, field, form) => onChange(options, monitorIdx, form),
+                      onBlur: (e, field, form) => onBlur(e, field, form),
+                      options: options,
+                      singleSelection: { asPlainText: true },
+                      selectedOptions: selection[monitorIdx] ? [selection[monitorIdx]] : undefined,
+                      'data-test-subj': `monitors_list_${monitorIdx}`,
+                      fullWidth: true,
+                    }}
+                  />
+                </EuiFlexItem>
+                {selection[monitorIdx] && (
+                  <EuiFlexItem
+                    grow={false}
+                    style={{
+                      margin: 0,
+                      padding: 0,
+                      width: '20px',
+                      height: '20px',
+                      position: 'absolute',
+                      right: '-20px',
+                      bottom: '24px',
+                    }}
+                  >
+                    <EuiToolTip content={'View monitor'}>
+                      <EuiButtonIcon
+                        aria-label={'View monitor'}
+                        iconType={'inspect'}
+                        color="text"
+                        target={'_blank'}
+                        href={`alerting#/monitors/${selection[monitorIdx].value}?alertState=ALL&from=0&monitorIds=${selection[monitorIdx].value}&search=&severityLevel=ALL&size=20&sortDirection=desc&sortField=start_time`}
+                      />
+                    </EuiToolTip>
+                  </EuiFlexItem>
+                )}
+                {fields.length > 2 && (
+                  <EuiFlexItem
+                    grow={false}
+                    style={{
+                      margin: 0,
+                      padding: 0,
+                      width: '20px',
+                      height: '20px',
+                      position: 'absolute',
+                      right: '-50px',
+                      bottom: '24px',
```

Comment:
Does this work well with different scaling screens?

Reply:
Let me fix this in follow up PR